ONSdigital/sdx-transform-cs
# Impute zero values for rsi and qbs

## What? and Why?
The RSI and QBS surveys are changing to route around the total breakdown questions (for turnover and employee numbers), meaning that SDX won't receive the q_codes for these answers. A new q_code will be present (d20 for turnover and d50 for employee numbers) when the total is zero and the breakdown questions have been routed around. As the downstream systems require answers for the breakdown questions this pull request imputes zero values for these q_codes when the new q_codes are present.

For QBS and RSI (0213 or 0215):
If d50 is present, 50, 51, 52, 53 and 54 are imputed with zero

For RSI:
If d20 is present, 20, 21, 22, 23, 24, 25 and 26 are imputed with zero

## How to test
- Fire up console and test variations of the above. Test json can be found in `tests/pck`:
  - 023.0203.turnover.total.zero.json
  - 023.0203.turnover.total.missing.json
  - 023.0213.employees.total.missing.json
  - 023.0213.employees.total.zero.json
  - 023.0213.totals.missing.json
  - 139.0001.missing.total.json
  - 139.0001.zero.total.json
- Check that the pck files have zero values for the imputed q_codes
- Check that the image doesn't display the imputed answers
